dynawo/sources/Models/Modelica/Dynawo/Examples/RVS/Components/StaticVarCompensators/BaseClasses/SVarCPVProp.mo

```diff
@@ -0,0 +1,60 @@
+within Dynawo.Examples.RVS.Components.StaticVarCompensators.BaseClasses;
```

Comment:
The SVarC of the same name in the library includes a regulation relying on a parameter Lambda. Maybe the RVS SVarcPVProp ought to be renamed.

Reply:
Look at SVarCPV then

Reply:
SVarCPV of the library also includes a calculation of BVarPu.

Reply:
SVarCPV of the library is used nowhere, nor is it extended by another model.

Using this SVarC instead of the RVS one in the test cases leads to simulation failures caused by the equation

BVarPu = 0;

applied if the SVarC is not running.

I suggest to modify the SVarC of the library in order to enable the RVS simulations. What do you think ?

Reply:
It is used in DFL so we keep it that way

Reply:
Agreed. I have added a model SVarCPVNoLimits to the library, as the susceptance limits do not apply in this model.